system/mp/mpbase.cpp

```diff
@@ -555,7 +555,12 @@ protected: friend class CNodeIterator;
     {
         SocketEndpointArray epa;
         epa.fromText(s,defport);
-        return createIGroup(epa);
+        ForEachItemIn(idx, epa)
+        {
+            if (!epa.item(idx).isNull())
```

Comment:
It's avoiding a delay in the case that someone specifies an invalid name for the dali. Most components don't care. I don't think it really needs another Jira.